README.md

```diff
@@ -90,17 +90,14 @@ Kops also has time set aside every other week to offer help and guidance to the
 community. Kops maintainers have agreed to set aside time specifically dedicated
 to working with newcomers, helping with PRs, and discussing new features.
 
-We recommend letting us know **beforehand** if you plan on attending so we can
-have time to prepare for the call.
+Please feel free to reach out **beforehand** if you plan on attending!
 
-| Maintainer   | Schedule      |  URL |
-|--------------|---------------|-------|
-| [@justinsb](https://github.com/justinsb)             |  2nd / 4th Friday 9am PDT | [Zoom](https://zoom.us/my/k8ssigaws) |
-| [@chrislovecnm](https://github.com/chrislovecnm)     |  2nd / 4th Friday 9am PDT | [Zoom](https://zoom.us/my/k8ssigaws) |
-| [@kris-nova](https://github.com/kris-nova)           |  2nd / 4th Friday 9am PDT | [Zoom](https://zoom.us/my/k8ssigaws) |
+Office hours, on [Zoom](https://zoom.us/my/k8ssigaws) video
+conference are on Fridays at 9 am PT every other week, on odd week numbers.
```

Comment:
This should mention the time in UTC (i.e. 17:00 / 5PM UTC) and contain a link to [this view](http://www.worldtimebuddy.com/?pl=1&lid=100,5,8,2950159&h=100) which highlights the most important EU/US timezones & UTC and gives users the ability to easily add their own local timezone to the view.